### PR TITLE
Tune for VGPU

### PR DESCRIPTION
**Pull request by andr2000**
Add required scripts, set correct build options and paths
to build PVR KM/UM with virtualization support.

Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>

### CODE REVIEW

**Comment by andr2000** [comment]
please merge

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-salvator-x-dom0.dts [hunk 1]
```diff
@@ -1101,22 +1101,6 @@
...
-			 <&ipmmu_pv1 6>, <&ipmmu_pv1 7>,
-			 <&ipmmu_pv2 0>, <&ipmmu_pv2 1>,
-			 <&ipmmu_pv2 2>, <&ipmmu_pv2 3>,
-			 <&ipmmu_pv2 4>, <&ipmmu_pv2 5>,
-			 <&ipmmu_pv2 6>, <&ipmmu_pv2 7>,
-			 <&ipmmu_pv3 0>, <&ipmmu_pv3 1>,
-			 <&ipmmu_pv3 2>, <&ipmmu_pv3 3>,
-			 <&ipmmu_pv3 4>, <&ipmmu_pv3 5>,
-			 <&ipmmu_pv3 6>, <&ipmmu_pv3 7>;
 };
```
Comment (otyshchenko1) [inline recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-salvator-x-dom0.dts]: Pls disable all ipmmu_pv modules all well. Here and for r8a7796-salvator-x-dom0.dts

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dump_fw_log.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+while true; do
+      echo "----------------------"
+#      cat /sys/kernel/debug/pvr/firmware_trace | grep "OSid = 1";
```
Comment (aanisov) [inline recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dump_fw_log.sh]: An odd line?

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/pvr_debug_setup.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+pvrdebug -log trace -loggroups main,mts,cleanup,csw,bif,pm,rtd,spm,pow,hwr,hwp,rpm,dma,debug
```
Comment (aanisov) [inline recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/pvr_debug_setup.sh]: Why the file named *sh has not #!/bin/sh?

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-salvator-x-dom0.dts (hunk 1 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-salvator-x-dom0.dts]: for what reason?

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb [hunk 4]
```diff
@@ -11,11 +11,17 @@ LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384
 
 SRC_URI = "\
   file://domu.cfg \
+  file://domc.cfg \
```
Comment (aanisov) [inline recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb]: Is this domc needed here?
If you are keeping it here, then commented lines in domu.cfg looks odd

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-salvator-x-dom0.dts (hunk 1 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-salvator-x-dom0.dts]: done

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/pvr_debug_setup.sh (hunk 3 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/pvr_debug_setup.sh]: default shell is used

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dump_fw_log.sh (hunk 2 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/dump_fw_log.sh]: not really, commented one greps for OSID 1 specifically

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb (hunk 4 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb]: it has different name (so displ_be can be setup properly), IP address and MAC

**Pull request merged**